src/index.html

```diff
@@ -7,12 +7,59 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      data-qa="card"
+      class="product"
+    >
+      <div class="main-image product__main-image"></div>
+
+      <p class="item item--name product__item--name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="item item--description product__item--description">
+        Product code: 195434
+      </p>
+
+      <div class="wrapper product__wrapper">
```

Comment:
BEM methodology suggests that modifiers should not be used alone. Here, 'product__wrapper' appears to be a block element rather than a modifier. Ensure you follow BEM naming conventions correctly.